resec/redis/new.go

```diff
@@ -29,8 +29,8 @@ func NewConnection(m *cli.Context) (*Manager, error) {
 			"redis_addr": m.String("redis-addr"),
 		}),
 		state:     &state.Redis{},
-		stateCh:   make(chan state.Redis, 10),
```

Comment:
I would undo these back to 10, as we discussed in a standup, this will help prevent blocking operations from the Redis loop in case the reconciler can't keep up